src/xmipp/bindings/python/python_image.cpp

```diff
@@ -34,9 +34,16 @@
 /*                            Image                         */
 /**************************************************************/
 
+ImageObject* createImageObject() {
+    auto *result = PyObject_New(ImageObject, &ImageType);
+    result->image.release(); // Mind the uninitialized memory
```

Comment:
We should not be doing this.
From the docs (https://docs.python.org/3/c-api/allocation.html) it seems that those fields should be properly initialized by the contructor.